cnd/src/db.rs

```diff
@@ -51,12 +51,18 @@ pub struct ForSwap<T> {
     pub data: T,
 }
 
-/// Load data from the database.
+/// Load data from the database for a particular swap id.
```

Comment:
Why did you remove this part of the comment? It seems even more relevant with the addition of `LoadAll`.

Reply:
Lol, I added this part of the comment :)

Reply:
ROFL, that's funny.